Userland/Libraries/LibCrypto/Curves/X25519.h

```diff
@@ -18,6 +18,7 @@ class X25519 : public EllipticCurve {
     ErrorOr<ByteBuffer> generate_public_key(ReadonlyBytes a) override;
     ErrorOr<ByteBuffer> compute_coordinate(ReadonlyBytes a, ReadonlyBytes b) override;
     ErrorOr<ByteBuffer> derive_premaster_key(ReadonlyBytes shared_point) override;
+    ErrorOr<bool> check_signature(ReadonlyBytes hash, ReadonlyBytes pubkey, ReadonlyBytes signature) override;
```

Comment:
X25519 and X448 are not used for signatures, only the ED version is used for that.